cpp/include/cudf/io_readers.hpp

```diff
@@ -282,4 +281,83 @@ class OrcReader {
   ~OrcReader();
 };
 
+/**---------------------------------------------------------------------------*
+ * @brief Options for the Parquet reader
+ *---------------------------------------------------------------------------**/
+struct ParquetReaderOptions {
```

Comment:
```suggestion
struct parquet_reader_options {
```

or

```suggestion
namespace parquet {

struct options {
```

or 

```suggestion
namespace parquet {
struct reader_options {
```

Reply:
Done! Good suggestion with namespaces, looks cleaner and more uniform!